packages/cadl-ranch-specs/http/lro/lro-basic/main.tsp

```diff
@@ -9,22 +9,35 @@ using Azure.Core;
 @doc("All the basic lro test shall be added in this file")
 namespace Azure.Lro;
 
+@doc("Details about a user.")
+model User {
+  @doc("The name of user.")
+  name: string;
+}
+
 #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "For testing"
 @scenario
-@scenarioDoc("The polling url is in operation-location of response headers. Mock api finally return 'Test for polling succeed'")
+@scenarioDoc("""
+Expected final response body:
+```json
+{
+  "name": "bob"
+}
+```
+""")
 @route("/put")
 interface PollingSuccess {
   @doc("Test for basic lro of put")
   @pollingOperation(PollingSuccess.polling)
   @put
-  create(): string;
+  create(): User;
 
   @doc("The polling url")
   @route("/polling")
   @get
-  polling(): string;
+  polling(): User;
```

Comment:
Not sure whether polling needs a `status` property.

Reply:
i think python's core will look at "Succeeded". I'm going to keep it in for now to unblock people trying to move to latest `getLroMetadata`, definitely down to revisit later. thanks!

Reply:
OK for me.

Java will need to convert it to User model (as signature is `PollerFlux<User, User>`). And as "name" is required in User, some place would fail (in the case response only have `status`.
Not exactly same issue as my original comment.